compiler/daml-extension/package.json

```diff
@@ -139,7 +139,10 @@
         "@types/which": "1.3.1",
         "@types/xml2js": "^0.4.5",
         "fp-ts": "^2.1.1",
+        "markdown-it": "^13.0.1",
+        "minimist": "^1.2.6",
         "node-fetch": "^2.6.7",
+        "simple-get": "^4.0.1",
```

Comment:
Are these valid? Do we need to explicitly add them to the dependencies list?

Reply:
No, you should not do this move. If you cannot simply remove from resolutions for #13383 without breaking `yarn audit`, you should just leave it where it is.

Reply:
Thanks @S11001001 , removed!